### PR TITLE
#631 deleted DolphinStage and DolphinWindow

### PR DESCRIPTION
**Pull request by joergsesterhenn**
Hi @hendrikebbers!
I just removed the two files since there was nothing of value that could have been pushed to the util class.
I wonder if it would make sense to remove the util class as well since it was only used by the two deleted classes.

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/680)

### CODE REVIEW

**Comment by joergsesterhenn** [comment]
Travis failed due to missing permissions to run sonarqube ... I guess that is not caused by my change.

**Comment by hendrikebbers** [comment]
No, that is a problem that we currently have. To check everything it is enough to do a 

gradlew clean build

on your local machine.

**Comment by hendrikebbers** [comment]
@joergsesterhenn as https://github.com/canoo/dolphin-platform/issues/631#issuecomment-337733298 says the functionality should be added to the DolphinWindowUtils class. This PR only removes the 2 classes and do not add the same functionally to the util class. This should be part of this PR.

**Comment by joergsesterhenn** [comment]
What functionality do you need? A createAutocloseStage() method that would return a Stage? Why not just delete the complete package? Seems not to be used...

**Comment by hendrikebbers** [comment]
@joergsesterhenn it is not used internally but it should be a help for people that use the framework ;)
And yes, the factory should have methods that return stage / window with the functionallity that was defined in the separate classes.

**Comment by joergsesterhenn** [comment]
There seems to be a cyclic dependency that prevents me from building the dolphin-platform-remoting-client-javafx:

A cycle was detected in the build path of project 'dolphin-platform-remoting-client-javafx'. The cycle consists of projects {dolphin-platform-remoting-client, dolphin-platform-remoting-server, dolphin-platform-remoting-client-javafx}

**Comment by joergsesterhenn** [comment]
Please have a look if that suits your needs.

File: platform/dolphin-platform-remoting-client-javafx/src/main/java/com/canoo/platform/remoting/client/javafx/window/DolphinWindowUtils.java [hunk 1]
```diff
@@ -43,4 +84,25 @@
         return () -> window.removeEventFilter(WindowEvent.WINDOW_HIDDEN, handler);
     }
 
+	/**
+	 * The method will create a JavaFX {@link Window} that contains the view of a {@link AbstractViewController} and will automatically call
+	 * {@link AbstractViewController#destroy()} when the stage becomes hidden.
+	 *
+	 * @param <M> type of the model
+	 */
+    public static <M> Window createWindow(AbstractViewController<M> viewBinder) {
```
Comment (hendrikebbers) [inline platform/dolphin-platform-remoting-client-javafx/src/main/java/com/canoo/platform/remoting/client/javafx/window/DolphinWindowUtils.java]: please add final keyword

File: platform/dolphin-platform-remoting-client-javafx/src/main/java/com/canoo/platform/remoting/client/javafx/window/DolphinWindowUtils.java [hunk 2]
```diff
@@ -43,4 +84,25 @@
...
+       Assert.requireNonNull(viewBinder, "viewBinder");
+	   return new DolphinWindow<M>(viewBinder);
+    }
+    
+	/**
+	 * The method will create a JavaFX {@link Stage} that contains the view of a {@link AbstractViewController} and will automatically
+	 * call {@link AbstractViewController#destroy()} when the stage becomes hidden.
+	 * @param <M> type of the model
+	 */
+    public static <M> Stage createStage(final AbstractViewController<M> viewBinder) {
```
Comment (hendrikebbers) [inline platform/dolphin-platform-remoting-client-javafx/src/main/java/com/canoo/platform/remoting/client/javafx/window/DolphinWindowUtils.java]: please add final keyword

**Comment by hendrikebbers** [comment]
@joergsesterhenn 👍 Only a minimum change is needed. Good work

**Comment by joergsesterhenn** [comment]
Damn I really need to improve my use of "final" ;)

**Comment by joergsesterhenn** [comment]
Finalized PR please have a look,

**Comment by hendrikebbers** [comment]
cool :) THX again!

**Pull request merged**